zipkin-lens/package.json

```diff
@@ -63,17 +63,21 @@
     "webpack-dev-server": "^3.1.14"
   },
   "dependencies": {
+    "@date-io/moment": "^1.3.7",
```

Comment:
`@date-io/moment` is necessary, when use `@material-ui/pickers` with `moment`.

Please see: https://material-ui-pickers.dev/getting-started/installation